### PR TITLE
changelog: 8.15.3 release

### PR DESCRIPTION
**Pull request by inge4pres**

### CODE REVIEW

**Review by lahsivjar — DISMISSED** [review]

File: changelogs/8.15.asciidoc [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 * <<apm-release-notes-8.15.1>>
 * <<apm-release-notes-8.15.0>>
 
+[float]
+[[apm-release-notes-8.15.3]]
+=== APM version 8.15.3
+
+https://github.com/elastic/apm-server/compare/v8.15.2\...v8.15.3[View commits]
+
+No significant changes.
```
[the remark this answers is not in the document]
Comment (inge4pres) [reply changelogs/8.15.asciidoc:15]: thanks Carson, I was looking at https://github.com/elastic/apm-server/compare/v8.15.2...update-8.15.3

Is it this? https://github.com/elastic/apm-server/pull/13950

[the remark this answers is not in the document]
Comment (inge4pres) [reply changelogs/8.15.asciidoc:15]: I can only see these changes in the version diffs, 1.10.0 to 1.11.0
https://github.com/elastic/apm-data/compare/v1.10.0...v1.11.0

Apparently only https://github.com/elastic/apm-server/issues/13703 should be added?

[the remark this answers is not in the document]
Comment (carsonip) [reply changelogs/8.15.asciidoc]: Should be https://github.com/elastic/apm-data/compare/v1.9.1...v1.11.0 if I'm looking at v8.15.2 correctly.
On top of #13703, there's https://github.com/elastic/apm-data/pull/339 . @lahsivjar any otel lib changes that are significant?

**Review by 1pkg — COMMENTED** [review]
We also merged https://github.com/elastic/apm-server/pull/13884 in 8.15.3, originally I didn't add any changelog record for this, but now I'm not sure if we should?

**Review by marclop — APPROVED** [review]

File: changelogs/8.15.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (inge4pres) [reply changelogs/8.15.asciidoc]: Yep, you are correct https://github.com/elastic/apm-server/blob/v8.15.2/go.mod#L10
Though I don't see apm-data PR 339 in https://github.com/elastic/apm-data/compare/v1.9.1...v1.11.0

**Comment by inge4pres** [comment]
> I'm not sure if we should?

Personally i vote for not including it, because it's not a user-visible change that impacts consuming the product.

File: changelogs/8.15.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (carsonip) [reply changelogs/8.15.asciidoc]: > Though I don't see apm-data PR 339 in https://github.com/elastic/apm-data/compare/v1.9.1...v1.11.0

Commit message was too long. It is the one from [Martijn Laarman](https://github.com/Mpdreamz)

File: changelogs/8.15.asciidoc [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+[float]
+[[apm-release-notes-8.15.3]]
+=== APM version 8.15.3
+
+https://github.com/elastic/apm-server/compare/v8.15.2\...v8.15.3[View commits]
+
+[float]
+==== Bug fixes
+
+- Fix a panic in OTLP intake protocol, occurring when invalid format of data was included in arrays {pull}13703[13703]
```
Comment (carsonip) [inline changelogs/8.15.asciidoc:18]:
```suggestion
- Fix a panic in OTLP label setting when receiving on non-compliant attribute array values {pull}13950[13950]

[float]
==== Added

- Map OpenTelemetry instrumentation scope to `Service.Framework.*` for all signal types. {pull}13903[13903]
```

File: changelogs/8.15.asciidoc [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+
 [float]
 [[apm-release-notes-8.15.2]]
 === APM version 8.15.2
```
Comment (carsonip) [inline changelogs/8.15.asciidoc:22]: Can you delete the relevant lines (30 and 35) in 8.15.2, they are moved to 8.15.3. I hate the github review UI.

**Review by carsonip — CHANGES_REQUESTED** [review]
sorry for being a PITA every time about changelogs. I realize I was wrong in https://github.com/elastic/apm-server/pull/14186 . The apm-data backport didn't make it on time, and there were 2 entries in 8.15.2 actually slipped into 8.15.3.

File: changelogs/8.15.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (inge4pres) [reply changelogs/8.15.asciidoc]: Apologies... I'll add

[the remark this answers is not in the document]
Comment (inge4pres) [reply changelogs/8.15.asciidoc]: mhmhm that change is already in 8.15.2 changelog entry, not sure why.
https://github.com/elastic/apm-server/blob/d218c50f62ce4e34f449c334517268aeed6f56fc/changelogs/8.15.asciidoc#L32-L35

**Comment by inge4pres** [comment]
> sorry for being a PITA every time about changelogs.

nw, actually thanks so much for your help on this 🙏🏼 
looking forward to automating away all these steps

**Review by carsonip — APPROVED** [review]

**Pull request merged**